requirements.txt

```diff
@@ -10,14 +10,14 @@ botocore==1.4.5
 cffi==1.15.1
 colorama==0.3.3
 configparser==3.5.0
-couchbase==3.2.7
+#couchbase==3.2.7
 cryptography==37.0.4
 docker==2.2.1
 docker-pycreds==0.2.1
 docutils==0.12
 ecdsa==0.13
 enum34==1.1.6
-flake8==3.9.2
+flake8==3.3.0
```

Comment:
I hope this doesn't break ios, c, and sync-gateway jobs. 
I will check it in my system. Approving as of now, will let you know if any error arises